src/azure-cli/azure/cli/command_modules/backup/custom_help.py

```diff
@@ -366,7 +366,7 @@ def get_operation_id_from_header(header):
 
 
 def get_vault_from_arm_id(arm_id):
-    m = re.search('(?<=vaults/)[^/]+', arm_id)
+    m = re.search('(?<=/vaults/)[^/]+', arm_id)
```

Comment:
May I ask why you add `/` here?

Reply:
In test_backup_wl_sql_restore_as_files the rg name we are using ends with 'vaults' like shown below
![image](https://user-images.githubusercontent.com/26112239/123036762-3131ad00-d40b-11eb-802f-a11f66885c5c.png)

The code was confusing this with the actual 'vaults' in the arm_id so I added a '/' before 'vaults'. Now the code searches for '/vault' so there's no confusion.

Reply:
Got it, thanks~